### ISSUE TITLE
Polygate and standard drum notation in Seq++

### ISSUE THREAD

**Issue by ajbmorris** [squinkylabs/SquinkyVCV#106]
Just playing with polygate to get a drum notation sequence out of Seq++.
it would be nice if it recognised the standard 13 "notes" from D4 to B6 somehow,  so for instance bass drum appears on E4, snare on C5 and plate smashing on B6 etc
![DRUM-KEY](https://user-images.githubusercontent.com/50493801/66256618-9a7d1280-e787-11e9-91c7-6eb58ceacbfe.jpg)
luckily I spotted the way through after starting with the bass drum LOL and will transpose accordingly for now...
BTW great modules!

**Comment by squinkylabs** [comment]
I have mixed feelings about this suggestion. On the positive side, it would for sure be nice to allow different pitches. On the negative side, I would guess you might be the only person in the world who wants to use VCV with standard drum notation ;-) Also, this would require more than 8 outputs. But will definitely consider something more versatile in the future.

**Comment by ajbmorris** [comment]
I perfectly understand, and the 8 outputs of polygate are fine for most things. I'm assuming that polygate takes the 8 semitones up from C4 in sequence from the piano roll of seq++?

**Comment by squinkylabs** [comment]
Yes, the semitones starting at C4 and going up. I just noticed there is no online documentation for Polygate, so I just entered a bug for that.
